### PR TITLE
Updated gym, styles for singleGym, added helper for handlebars

### PR DESCRIPTION
**Pull request by aramjee**

### CODE REVIEW

File: middleware.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { userData } from './data/index.js';
+
+const middleware = {
```
Comment (XinxuanLyu) [inline middleware.js:3]: Not sure if these functions can be considered as middleware. These functions did not impact the current transaction or the next middleware. These functions just simply check if the user is logged in or is a gym owner. 
Should belongs to the helper method. If you want to add a function in middleware, you can make changes in app.js. There is one template, which can be used for logging feature.

**Review by XinxuanLyu — CHANGES_REQUESTED** [review]
Please check the comment in middleware.js

File: middleware.js (hunk 1 above)
Comment (chloecck) [reply middleware.js:3]: I agree with Xinxuan. Middleware should be something that's happen before it hit the routes, and these two functions checkIfLoggedIn and checkIfGymOwner, even though with similar results, resides within the routes thus is a helper/validation method. 

But we can reduce the duplicate codes in the middleware inside app.js by directly calling these two methods?

File: helpers.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { userData } from './data/index.js';
```
Comment (XinxuanLyu) [inline helpers.js:1]: It is OK if you accidentally remove the file during rename. Maybe move the helpers.js and middleware.js into the public folder

**Review by XinxuanLyu — CHANGES_REQUESTED** [review]

File: helpers.js (hunk 2 above)
Comment (XinxuanLyu) [reply helpers.js:1]: Otherwise LGTM

**Pull request merged**